app/src/CoffeeShopApp.kt

```diff
@@ -0,0 +1,33 @@
+import java.io.File
+import java.io.FileNotFoundException
+
+class CoffeeShopApp (val x: Double, val y: Double, val filename: String) {
+    fun getNearestShops(): List<CoffeeShopWithDistance> {
```

Comment:
This does not make sens to be public, you should limit the public surface, read more about it `Indecent Exposure` code smell.

Reply:
I forgot about this. Will update the code.